src/coreclr/vm/codeman.cpp

```diff
@@ -4164,7 +4170,7 @@ void CodeHeader::EnumMemoryRegions(CLRDataEnumMemoryFlags flags, IJitManager* pJ
 // Enumerate for minidumps.
 //-----------------------------------------------------------------------------
 template<typename TCodeHeader>
-void EECodeGenManager::EnumMemoryRegionsForMethodDebugInfoWorker(CLRDataEnumMemoryFlags flags, MethodDesc * pMD)
+void EECodeGenManager::EnumMemoryRegionsForMethodDebugInfoWorker(CLRDataEnumMemoryFlags flags, EECodeInfo * pCodeInfo)
```

Comment:
Do we have any guarantees that pCodeInfo will be non-NULL entering this function?

Reply:
I see it's implicitly guaranteed in the caller's caller further down in the diff